### PR TITLE
an attempted react ssr

### PR DESCRIPTION
**Pull request by ScottDowne**

### CODE REVIEW

**Comment by ScottDowne** [comment]
I don't think this is good enough to merge, but it's kinda interesting?

File: components/link.jsx [hunk 1]
```diff
@@ -0,0 +1,13 @@
+var React = require('react');
+var Router = require('react-router');
+var Link = Router.Link;
+
+var Linker = React.createClass({
```
Comment (alicoding) [inline components/link.jsx]: Do we need this now then?

Comment (ScottDowne) [inline components/link.jsx]: Yeah, I hate remembering how to get to Link every time I need it, which is often, usually requiring me to open up another .jsx file that uses Link and copying and pasting.

Also, I mean to modify this to swap between <a href=""> and <Link to=""> depending on if the link is internal or external. Something like this: https://github.com/mozilla/teach.webmaker.org/blob/develop/components/icon-link.jsx#L6-L22

**Comment by ScottDowne** [comment]
R? This is ready to land, imo.

I would rather land it than keep working on it.

I know there are two demo pages in there that we don't need, but I'll just convert them to sequential and thank you page paths when those pages are made into react.

**Comment by alicoding** [comment]
We will improve this later, but lets land and iterate through this.

**Pull request merged**